src/components/category-selection/category-badge/CategoryBadge.module.scss

```diff
@@ -1,4 +1,8 @@
+@use '../../../styles/config/mixins' as *;
+
 .categoryBadge {
+    @include flex;
+    gap: 10px;
     border-radius: var(--border-radius-outer);
     padding: var(--spacing-xs) var(--spacing-sm);
     font-weight: var(--font-weight-medium);
```

Comment:
Since the badge may now accommodate two children, please create distinct styles for each child. The parent element should manage its own style and solely handle the positioning/layout, without styling its children. This methodology, consistently applied throughout the project, simplifies debugging later on.